tests/core/test_auth.py

```diff
@@ -0,0 +1,48 @@
+import filecmp
+import json
+import os
+from pathlib import Path
+
+import eth_keyfile
+from eth_utils import is_same_address, to_text
+import pytest
+
+from ethpm_cli.auth import (
+    get_authorized_address,
+    get_authorized_private_key,
+    import_keyfile,
+)
+from ethpm_cli.constants import KEYFILE_PATH
+
+PRIVATE_KEY = b"\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01\x01"  # noqa: E501
+ADDRESS = "0x1a642f0E3c3aF545E7AcBD38b07251B3990914F1"
+PASSWORD = b"password"
+KEYFILE_JSON = eth_keyfile.create_keyfile_json(PRIVATE_KEY, PASSWORD)
+
+
+@pytest.fixture
+def keyfile(monkeypatch, tmpdir):
+    monkeypatch.chdir(tmpdir)
+    ethpmcli_dir = Path(tmpdir / "ethpmcli_dir")
+    ethpmcli_dir.mkdir()
+    monkeypatch.setenv("XDG_ETHPMCLI_ROOT", str(ethpmcli_dir))
```

Comment:
This is probably something that you want to be in the root `conftest.py` file and to flag it with `autouse=True` **AND** to have a test that sanity checks that none of the utils that fetch this directory return the *real* directory so that you have strong confidence that you aren't accidentally modifying the actual user director during test runs.